build.js

```diff
@@ -87,7 +109,7 @@ function ensureNoChangesOnMasterBeforePublish() {
 const buildScriptDirectory = path.dirname(process.argv[1]);
 shell.cd(path.join(buildScriptDirectory));
 
-ensureNoChangesOnMasterBeforePublish();
+//ensureNoChangesOnMasterBeforePublish();
```

Comment:
I'm assuming you didn't mean to comment this out.

Reply:
Yikes - I did that to test and thought I had gotten it uncommented.